test/e2e/configmap.go

```diff
@@ -181,7 +181,7 @@ var _ = Describe("ConfigMap", func() {
 		// Kubelet projects the update into the volume and the container picks
 		// it up. This timeout is based on the default Kubelet sync period (1
 		// minute) plus additional time for fudge factor.
-		const podLogTimeout = 90 * time.Second
+		const podLogTimeout = 150 * time.Second
```

Comment:
I think there are too many tests that poll for 5m in gce normal. I think we can catch if this just keeps crossing 5m via the test dashboard, I don't think any of the tests that currently poll for 5m actually do. If one of the generic timesout in e2e/util are applicable in this case, just use that and we'll bump all at the same time if we do?